openspp-project/openspp-registry
# Fix Registry Icon in SPP Muk Theme

## **Why is this change needed?**

There is a missing icon in the Registry menu

## **How was the change implemented?**

Added an icon for Registry Menu

## **New unit tests**

No unit test

## **Unit tests executed by the author**

No unit test

## **How to test manually**

- Install `OpenSPP Theme (Muk Theme)`
- click the `9 squares` logo on the most top-left part of the screen
- Check if Registry have icon now.

## **Related links**
https://github.com/OpenSPP/openspp-modules/issues/524
